packages/synthetic-chain/src/cli/dockerfileGen.ts

```diff
@@ -40,15 +35,16 @@ RUN /usr/src/upgrade-test-scripts/start_ag0.sh
 `;
   },
   /**
-   * Resume from latest production state
+   * Resume from state of an existing image
+   * @param fromTag
    * @param proposalName
    * @param to
    */
-  RESUME(proposalName: string, to: string) {
+  RESUME(fromTag: string, proposalName: string, to: string) {
     return `
 ## RESUME
 # on a3p base, with upgrade to ${to}
-FROM ${baseImage} as prepare-${proposalName}
+FROM ghcr.io/agoric/agoric-3-proposals:${fromTag} as prepare-${proposalName}
```

Comment:
This makes me realize, does it make sense to hard code the base image name in the synthetic-chain tool?

Reply:
I vacillated on this. For now I think it's reasonable, for ergonomics. It's the only chain right now. The code is factored such that it will be easy to accommodate other chains when needed.